frame/assets/src/functions.rs

```diff
@@ -253,6 +282,48 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		Ok((credit, maybe_burn))
 	}
 
+	/// Creates a account for `who` to hold asset `id` with a zero balance and takes a deposit.
+	pub(super) fn do_touch(id: T::AssetId, who: T::AccountId) -> DispatchResult {
+		ensure!(!Account::<T, I>::contains_key(id, &who), Error::<T, I>::AlreadyExists);
+		let deposit = T::AssetAccountDeposit::get();
+		let mut details = Asset::<T, I>::get(&id).ok_or(Error::<T, I>::Unknown)?;
+		let reason = Self::new_account(&who, &mut details, Some(deposit))?;
+		T::Currency::reserve(&who, deposit)?;
+		Asset::<T, I>::insert(&id, details);
+		Account::<T, I>::insert(
+			id,
+			&who,
+			AssetAccountOf::<T, I> {
+				balance: Zero::zero(),
+				is_frozen: false,
+				reason,
+				extra: T::Extra::default(),
+			},
+		);
+		Ok(())
+	}
+
+	/// Returns a deposit, destroying an asset-account.
+	pub(super) fn do_refund(id: T::AssetId, who: T::AccountId, allow_burn: bool) -> DispatchResult {
+		let mut account = Account::<T, I>::get(id, &who).ok_or(Error::<T, I>::NoDeposit)?;
+		let deposit = account.reason.take_deposit().ok_or(Error::<T, I>::NoDeposit)?;
+		let mut details = Asset::<T, I>::get(&id).ok_or(Error::<T, I>::Unknown)?;
+
+		ensure!(account.balance.is_zero() || allow_burn, Error::<T, I>::WouldBurn);
```

Comment:
the doc of the call is:

> 		/// - `allow_burn`: If `true` then assets may be destroyed in order to complete the refund.
>		/// This will be needed unless the asset is a provider and the balance is greater than the
>		/// asset's minimum balance.

is the provider/balance greater than minimum case supported ?

Reply:
No - docs updated.